doc/maintaining/installing/install-from-source.rst

```diff
@@ -5,10 +5,10 @@ Installing CKAN from source
 ===========================
 
 This section describes how to install CKAN from source. Although
-:doc:`install-from-package` is simpler, it requires Ubuntu 14.04 64-bit or
-Ubuntu 12.04 64-bit. Installing CKAN from source works with other versions of
-Ubuntu and with other operating systems (e.g. RedHat, Fedora, CentOS, OS X).
-If you install CKAN from source on your own operating system, please share your
+:doc:`install-from-package` is simpler, it requires Ubuntu 16.04 64-bit or
```

Comment:
Good spot, I'll change that text back to "it requires Ubuntu 14.04 64-bit or
Ubuntu 12.04 64-bit" so it is consistent for the time being. I'm not sure if I'll get a chance to look at the packages too...